gradle/libs.versions.toml

```diff
@@ -1,31 +1,31 @@
 [versions]
-compileSdk = "32"
-targetSdk = "32"
+compileSdk = "33"
+targetSdk = "33"
 minSdk = "24"
 
 accompanist = "0.25.1"
-activityCompose = "1.5.1"
-amplify = "1.37.1"
+activityCompose = "1.6.0-rc02"
```

Comment:
As I remember we agreed don't use RC dependencies and modify filtering logic (remove "RC")
https://github.com/AnimealProject/animeal_android/blob/bc88312b1b7fe36fda389c30c61abe2b9ac11da3/build.gradle.kts#L72